doc/languages-frameworks/php.section.md

```diff
@@ -283,7 +283,10 @@ in {
   ];
 
   composerRepository = php.mkComposerRepository {
-    inherit (finalAttrs) src;
+    inherit (finalAttrs) pname version src;
+    composerNoDev = true;
+    composerNoPlugins = true;
+    composerNoScripts = true;
```

Comment:
I believe these are set to `true` by default, is it worth mentioning it here?

Reply:
They are not, 3eb168da9243989a63dfd4e2c17c52f8c133247b added assertions requiring them to be explicitly specified.

Reply:
mmh I do not remember that. Maybe it's better to require them explicitly when `mkComposerRepository` is used alone.

Reply:
I think implicit values should be fine for this, we would just have to choose the defaults. Probably either mirror `composer install` to align with expectations, or all `true` since missing things are typically easier to notice than bloat.